MAINTAINERS.md

```diff
@@ -77,8 +87,66 @@ Make frequent project releases to the community.
 
 ### Promote Other Maintainers
 
-Assist, add, and remove [MAINTAINERS](MAINTAINERS.md). Exercise good judgement, and propose high quality contributors to become co-maintainers.
+Assist, add, and remove [MAINTAINERS](MAINTAINERS.md). Exercise good judgement, and propose high quality contributors to become co-maintainers. See [Becoming a Maintainer](#becoming-a-maintainer) for more information.
 
 ### Describe the Repo
 
 Make sure the repo has a well-written, accurate, and complete description. See [opensearch-project/.github#38](https://github.com/opensearch-project/.github/issues/38) for some helpful tips to describe your repo.
+
+## Becoming a Maintainer
+
+You can become a maintainer by actively [contributing](CONTRIBUTING.md) to any project, and being nominated by an existing maintainer.
+
+### Nomination
+
+Any current maintainer starts an e-mail thread (until we have a better mechanism, e-mails can usually be found via MAINTAINERS.md + DCO) with all other maintainers on that repository to discuss nomination using the template below. In order to be approved, at least 3 positive maintainer votes are necessary, and no vetoes. In rare cases when there are less than 3 maintainers, the positive votes from all maintainers are required. Any disagreements can be escalated to the repo admin.
+
+The nomination should clearly identify the person with their real name and a link to their GitHub profile, and the rationale for the nomination, with concrete example contributions.
+
+### Interest
+
+Upon receiving at least 3 positive maintainer votes, and no vetoes, from other maintainers within a maximum period of 2 weeks, the nominating maintainer asks a potential nominee whether they might be interested in becoming a maintainer on the repository, which typically happens via e-mail obtained via DCO, but may also be a comment to a pull request, following a substantial contribution.
+
+> This is great work! Based on your ongoing contributions to the project we have decided that we'd like to invite you to become a co-maintainer for this project. Please let us know whether you accept this nomination.
+
+Individuals accept the nomination by replying, or commenting, for example _"Thank you! I would love to."_
+
+### Public Confirmation
+
+Upon receiving 3 positive maintainer votes, and no vetoes, from other maintainers, and after having confirmed interest with the nominee, the maintainer opens a pull request adding the proposed co-maintainer to MAINTAINERS.md. This makes the vote visible to the community. 
```

Comment:
Do you have any proposed language?

Reply:
I ask in part because I'm not sure what the exit requirements are here.

But, I'll give is a stab based on what I've gathered. If I understand what the overall process correctly, I'd modify the "Maintainer Decision" section as follows:

> At the end of two weeks, the maintainers will perform a final vote similar to the nomination process. If the nomination still has three positive votes, and no vetoes, from other maintainers, the nomination is accepted. The positive maintainer voters should approve the PR to show support. The PR may be merged with only one approver.

> If the maintainers cannot reach a positive decision for any reason, the nomination is rejected with "a decision could not be reached" with no further explanation necessary, and the pull request is closed.

Reply:
Ok, I see, this is clearer. Check out https://github.com/opensearch-project/.github/pull/59/commits/624799078f9413b1fa0f5a0122c0e731cc338c23.